custom_components/daikin_onecta/update.py

```
@@ -0,0 +1,167 @@
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Daikin update entities from a config entry."""
+    onecta_data: OnectaRuntimeData = config_entry.runtime_data
+    coordinator = onecta_data.coordinator
+
+    entities: list[DaikinFirmwareUpdateEntity] = []
+
+    for dev_id, device in onecta_data.devices.items():
```

Ruff (0.15.12)

[warning] Loop control variable `dev_id` not used within loop body

Rename unused `dev_id` to `_dev_id`

(B007)